packages/client/src/Tests/ActionsAndReducers/playerUI/handleVisibilityReducer.test.js

```diff
@@ -0,0 +1,98 @@
+/* global it expect */
+import handleVisibilityChanges from '../../../ActionsAndReducers/playerUi/helpers/handleVisibilityChanges'
+import findAsset from '../../../Components/Mapping/helpers/findAsset'
+
+const payload = [
+  {
+    assetId: 'C06',
+    by: 'Red',
+    newVis: true
+  }
+]
+
+const payload2 = [
+  {
+    assetId: 'C05',
+    by: 'Blue',
+    newVis: false
+  }, {
+    assetId: 'C05',
+    by: 'Red',
+    newVis: true
+  }
+]
+
+const allForces = [
```

Comment:
This JSON data shouldn't be in here, it should be in a separate file and referenced in any files that use it. Even if you only use data like this in one file, it's still good practice to separate it out, that way it becomes reusable code.

Reply:
Aah @foxleigh81  , I was worried about a new test requiring a JSON change that would break an existing test. But, since all the tests get run, the dev will find out quickly enough if this happens. Great.